src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -104,28 +91,25 @@ class Gold_Data() : Entity {
         this.flg = flg
     }
 
-    override fun isSet(): Boolean {
-        return _numSet && _txtSet && _lnkSet && _flgSet
+    fun copy(
+        num: Double = this.num,
+        txt: String = this.txt,
+        lnk: String = this.lnk,
+        flg: Boolean = this.flg
+    ) : Gold_Data {
+      return Gold_Data(
```

Comment:
I feel stupid nitpicking style for a generated file... But I'm going to do it anyway :stuck_out_tongue: 

```
fun copy(
    // args...
) = Gold_Data(
    // args...
)
```